docs: Update Studio/HD documentation with latest details (#20209)
unoplatform/uno: doc/articles/studio/Hot Design/hot-design-toolbox.md

When the search box is empty, the Toolbox displays all available controls organized into collapsible categories. These include standard WinUI 3 controls, layout containers, collection views, and any custom or third-party controls available in your project.

- **Common**: Frequently used WinUI 3 controls such as `Button`, `TextBox`, `CheckBox`, and `TextBlock`.
- **Layout**: Layout elements like `Grid`, `StackPanel`, and `Border` are used to structure the visual hierarchy of your XAML UI.
- **Collections**: Data-bound controls like `ListView`, `GridView`, and `ItemsRepeater`, typically used to present lists or repeated content.
- **Project**: Custom user controls and components defined in your current project or solution.
- **Custom**: Third-party controls, grouped by their assembly name.
- **All**: A complete, ungrouped list of all available controls and elements in the Toolbox.

Click the arrow beside a category name to expand or collapse its contents.

<img src="Assets/toolbox-expand-section.gif" height="400" alt="How to expand and collapse sections in the Toolbox." />

## Add a Control to the Canvas

To insert a control into your layout in the interactive **Canvas**:

> 1. Drag a control from the **Toolbox** window.

Reviewer: ```suggestion
1. Drag a control from the **Toolbox** panel.
```

2. Drop it onto the **Canvas**, inside the element where you want it to appear.

<img src="Assets/toolbox-add-to-canvas.gif" height="600" alt="How to drag and drop an element from the Toolbox window to the Canvas." />

## Add a Control to the Visual Tree

To insert a control into the **Elements** window:

> 1. Drag a control from the **Toolbox** window.
> 2. Drop it into the desired parent node in the visual tree inside the **Elements** window.

Reviewer: ```suggestion
1. Drag a control from the **Toolbox** panel.
2. Drop it into the desired parent node in the visual tree inside the **Elements** panel.
```

> <img src="Assets/toolbox-add-to-tree.gif" height="600" alt="How to drag and drop a control from the Toolbox window into the Elements window." />

Reviewer: ```suggestion
<img src="Assets/toolbox-add-to-tree.gif" height="600" alt="How to drag and drop a control from the Toolbox window into the Elements panel." />
```

## Insert a Control Using Double-Click

To quickly insert a control:

> 1. Select a parent element on the **Canvas** or in the visual tree in the **Elements** window.
> 2. Double-click the wanted new control in the **Toolbox**. It will be added as a child of the previously selected element.

Reviewer: ```suggestion
1. Select a parent element on the **Canvas** or in the visual tree in the **Elements** panel.
2. Double-click the control you want in the **Toolbox**. It will be added as a child of the currently selected element.
```

> 1. Select a parent element on the **Canvas** or in the visual tree in the **Elements** window.
> 2. Double-click the wanted new control in the **Toolbox**. It will be added as a child of the previously selected element.
> 

Reviewer: ```suggestion
1. Select a parent element on the **Canvas** or in the visual tree in the **Elements** panel.
2. Double-click the wanted new control in the **Toolbox**. It will be added as a child of the previously selected element.

```
